### PR TITLE
feat(icons): implement changes from v3.1.0

### PR DESCRIPTION
**Pull request by mivaylo**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] If applicable, have a visual design approval

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] clarity.design website / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: CDE-1985

## Does this PR introduce a breaking change?

- [x] Yes
- [ ] No

### CODE REVIEW

**Review by Jinnie — CHANGES_REQUESTED** [review]
1. Join all strings that are composed of smaller concatenated chunks.
2. Remove all leading spaces in the value strings.
3. Remove all fill="?????" occurrences. We should not impose color.
4. Raising the bundle size limits may be necessary but a full order of magnitude is a bit too much.

File: projects/core/src/icon/shapes/accessibility-2.ts [hunk 1]
```diff
@@ -9,9 +9,9 @@ import { IconShapeTuple } from '../interfaces/icon.interfaces.js';
...
+    '<path fill-rule="evenodd" clip-rule="evenodd" d="M18.0001 10C20.2092 10 22.0001 8.20914 22.0001 6C22.0001 3.79086 20.2092 2 18.0001 2C15.7909 2 14.0001 3.79086 14.0001 6C14.0001 8.20914 15.7909 10 18.0001 10ZM6.06006 11H30.0601C30.6123 11 31.0601 11.4477 31.0601 12C31.0601 12.5523 30.6123 13 30.0601 13H22.0001V22.68L25.7701 32.65C25.8642 32.898 25.8561 33.1732 25.7473 33.4151C25.6385 33.6571 25.4381 33.8459 25.1901 33.94C25.0741 33.9794 24.9525 33.9996 24.8301 34C24.4152 33.9972 24.0452 33.7386 23.9001 33.35L20.3801 24H15.6201L12.1201 33.35C11.9749 33.7386 11.6049 33.9972 11.1901 34C11.0676 33.9996 10.946 33.9794 10.8301 33.94C10.5821 33.8459 10.3816 33.6571 10.2728 33.4151C10.1641 33.1732 10.1559 32.898 10.2501 32.65L14.0001 22.65V13H6.06006C5.50777 13 5.06006 12.5523 5.06006 12C5.06006 11.4477 5.50777 11 6.06006 11ZM20.0001 22V13H16.0001V22H20.0001ZM20.0001 6C20.0001 4.89543 19.1046 4 18.0001 4C16.8955 4 16.0001 4.89543 16.0001 6C16.0001 7.10457 16.8955 8 18.0001 8C19.1046 8 20.0001 7.10457 20.0001 6Z" />',
```
Comment (kevinbuhmann) [inline projects/core/src/icon/shapes/accessibility-2.ts:12]:
```suggestion
    '<path d="M18.0001 10C20.2092 10 22.0001 8.20914 22.0001 6C22.0001 3.79086 20.2092 2 18.0001 2C15.7909 2 14.0001 3.79086 14.0001 6C14.0001 8.20914 15.7909 10 18.0001 10ZM6.06006 11H30.0601C30.6123 11 31.0601 11.4477 31.0601 12C31.0601 12.5523 30.6123 13 30.0601 13H22.0001V22.68L25.7701 32.65C25.8642 32.898 25.8561 33.1732 25.7473 33.4151C25.6385 33.6571 25.4381 33.8459 25.1901 33.94C25.0741 33.9794 24.9525 33.9996 24.8301 34C24.4152 33.9972 24.0452 33.7386 23.9001 33.35L20.3801 24H15.6201L12.1201 33.35C11.9749 33.7386 11.6049 33.9972 11.1901 34C11.0676 33.9996 10.946 33.9794 10.8301 33.94C10.5821 33.8459 10.3816 33.6571 10.2728 33.4151C10.1641 33.1732 10.1559 32.898 10.2501 32.65L14.0001 22.65V13H6.06006C5.50777 13 5.06006 12.5523 5.06006 12C5.06006 11.4477 5.50777 11 6.06006 11ZM20.0001 22V13H16.0001V22H20.0001ZM20.0001 6C20.0001 4.89543 19.1046 4 18.0001 4C16.8955 4 16.0001 4.89543 16.0001 6C16.0001 7.10457 16.8955 8 18.0001 8C19.1046 8 20.0001 7.10457 20.0001 6Z" />',
```

I don't think `fill-rule` or `clip-rule` are needed. Please check all.

**Review by Jinnie — COMMENTED** [review]
Just two very small suggestions:
- You can ceil values of bundle limits to first decimal, as it was before the change. x.xxxx -> x.x
- We can save some bytes by removing the spaces before />
As it was before the change.

File: projects/core/src/icon/shapes/accessibility-2.ts (hunk 1 above)
Comment (mivaylo) [reply projects/core/src/icon/shapes/accessibility-2.ts]: This is what we observe (we can see on the sixth icon - solid badged with danger) when we don't use the `clip-rule`:

image

Comment (mivaylo) [reply projects/core/src/icon/shapes/accessibility-2.ts]: I actually thought they are not needed in the beginning, so I had the frist three shapes with removed fill-rule and clip-rule properties, here is the commit I am adding them  back:

https://github.com/vmware-clarity/core/pull/323/commits/f9922ef78b4750b02812f19a389313cbab9a169c

**Review by Jinnie — APPROVED** [review]
Thank you!

**Pull request merged**